jdav_web/contrib/admin.py

```diff
@@ -107,7 +108,74 @@ def get_queryset(self, request):
 
 #class ObjectPermissionsInlineModelAdminMixin(rules.contrib.admin.ObjectPermissionsInlineModelAdminMixin):
 
-class CommonAdminMixin(FieldPermissionsAdminMixin, ChangeViewAdminMixin, FilteredQuerysetAdminMixin):
+class FieldCustomizationMixin:
+    @property
+    def field_key(self):
+        """returns the key to look if model has custom fields in settings"""
+        return f"{self.model._meta.app_label}_{self.model.__name__}".lower()
+
+    def get_excluded_fields(self):
+        """if model has custom excluded fields in settings, return them as list"""
+        return settings.CUSTOM_MODEL_FIELDS.get(self.field_key, {}).get("exclude", [])
+
+    def get_included_fields(self):
+        """if model has an entire fieldset in settings, return them as list"""
+        return settings.CUSTOM_MODEL_FIELDS.get(self.field_key, {}).get("fields", [])
```

Comment:
```suggestion
        return settings.CUSTOM_MODEL_FIELDS.get(self.field_key, {}).get("fields", None)
```
Required for the change below. The same should be done for the `exclude` variant.